app/src/features/surveys/view/survey-animals/animal.ts

```diff
@@ -6,6 +6,8 @@ import { v4 } from 'uuid';
 import { AnyObjectSchema, InferType, reach } from 'yup';
 import { AnimalTelemetryDeviceSchema } from './device';
 
+export const MANAGE_ANIMALS_DEFAULT_URL_PARAM = '0';
```

Comment:
Is there a lot of value in tracking this as a constant?

Reply:
Removed not needed anymore